docker/proof-fri-compressor/Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN curl https://sh.rustup.rs -sSf | bash -s -- -y && \
 WORKDIR /usr/src/zksync
 COPY . .
 
-RUN cargo build --release
+RUN cargo build --release --bin proof_fri_compressor
```

Comment:
Probably won't really improve stuff right now, since we [generate a bogus workspace in CI for prover images](https://github.com/matter-labs/zksync-era/blob/b0721f0641861fe879fd79c00dcd614497f6df35/.github/workflows/build-prover-template.yml#L163) that only contains one crate, but still a nice addition for the future.

Reply:
ah, so that's how it works..

as I kept wondering why we don't change the directory to 'prover'..